Makefile.dep

```diff
@@ -562,6 +562,22 @@ ifneq (,$(filter random,$(USEMODULE)))
     endif
 endif
 
+ifneq (,$(filter openthread,$(USEMODULE)))
+  USEMODULE += openthread_contrib
+  USEMODULE += libopenthread
+  USEMODULE += libmbedcrypto
+  USEPKG += openthread
+  USEMODULE+=xtimer
+
+  FEATURES_REQUIRED += cpp
+  FEATURES_REQUIRED += periph_spi
```

Comment:
I still don't understand why spi and gpio features are required. They should be already included by the radio driver.